test/Core.Test/Services/OrganizationServiceTests.cs

```diff
@@ -1831,4 +1832,31 @@ public void ValidateSecretsManagerPlan_ThrowsException_WhenMoreSeatsThanPassword
 
         sutProvider.Sut.ValidateSecretsManagerPlan(plan, signup);
     }
+
+    [Theory]
+    [EphemeralDataProtectionAutoData]
+    public async Task AcceptUserAsync_Success([OrganizationUser(OrganizationUserStatusType.Invited)] OrganizationUser organizationUser,
+        User user, SutProvider<OrganizationService> sutProvider)
+    {
+        user.Email = organizationUser.Email;
+
+        var dataProtector = sutProvider.GetDependency<IDataProtectionProvider>()
+            .CreateProtector("OrganizationServiceDataProtector");
+        // Token matching the format used in OrganizationService.InviteUserAsync
+        var token = dataProtector.Protect(
+            $"OrganizationUserInvite {organizationUser.Id} {organizationUser.Email} {CoreHelpers.ToEpocMilliseconds(DateTime.UtcNow)}");
+        var userService = Substitute.For<IUserService>();
+
+        sutProvider.GetDependency<IGlobalSettings>().OrganizationInviteExpirationHours.Returns(24);
```

Comment:
This seems useful, is there any test helper class we can put it in for future use?

Reply:
I didn't find an appropriate place. In the future if we reuse this elsewhere I'll add it to `Core.Utilities`.